authsnkr/src/js/snkrcertificate.js

```diff
@@ -22,10 +22,9 @@ function snkrImgGen(image_url){
 function addSneakersToPage(list, $id){
     $id.empty();
     var $snkrDiv;
-    
-    for (var i=0; i<list.length; i++){
-        $snkrDiv = createSneakerDiv(list[i][0], list[i][1]["c"][0], list[i][3]["c"][0], list[i][4]["c"][0]);
-        
+    debugger;
```

Comment:
can we remove this debugger

Reply:
Thanks for pointing that out @richkuo. I took it out.